CIP-0013/README.md

```diff
@@ -185,11 +185,15 @@ For a CIP based on this principle, see [CIP-0017](https://github.com/cardano-fou
 
 - [x] There exist one or more wallets supporting Payment URIs.
   - [x] Yoroi
+  - [x] beginWallet
```

Comment:
```suggestion
  - [x] Begin Wallet
```
On the web site it's spelled consistently either `Begin Wallet` or simply `Begin`.